mmv1/products/networkconnectivity/RegionalEndpoint.yaml

```diff
@@ -0,0 +1,126 @@
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'network_connectivity_regional_endpoint_regional_access'
+    primary_resource_id: 'default'
+    vars:
+      rep_name: 'my-rep'
+      rep_network: 'my-network'
+      rep_subnetwork: 'my-subnetwork'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'network_connectivity_regional_endpoint_global_access'
+    primary_resource_id: 'default'
+    vars:
+      rep_name: 'my-rep'
+      rep_network: 'my-network'
+      rep_subnetwork: 'my-subnetwork'
+
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    immutable: true
+    url_param_only: true
+    description: |
+      The name of the RegionalEndpoint.
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The location of the RegionalEndpoint.
+
+properties:
+  - !ruby/object:Api::Type::Time
+    name: 'createTime'
+    output: true
+    description: |
+      Time when the RegionalEndpoint was created.
+  - !ruby/object:Api::Type::Time
+    name: 'updateTime'
+    output: true
+    description: |
+      Time when the RegionalEndpoint was updated.
+  - !ruby/object:Api::Type::KeyValueLabels
+    name: 'labels'
+    description: |
+      User-defined labels.
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      A description of this resource.
+  - !ruby/object:Api::Type::String
+    name: 'targetGoogleApi'
+    required: true
+    description: |
+      The service endpoint this private regional endpoint connects to. Format: `{apiname}.{region}.p.rep.googleapis.com` Example: \"cloudkms.us-central1.p.rep.googleapis.com\".
+  - !ruby/object:Api::Type::String
+    name: 'network'
+    description: |
+      The name of the VPC network for this private regional endpoint. Format: `projects/{project}/global/networks/{network}`
+    default_from_api: true
+  - !ruby/object:Api::Type::String
+    name: 'subnetwork'
+    description: |
+      The name of the subnetwork from which the IP address will be allocated. Format: `projects/{project}/regions/{region}/subnetworks/{subnetwork}`
+    default_from_api: true
+  - !ruby/object:Api::Type::Enum
+    name: accessType
+    required: true
+    description: |
+      The access type of this regional endpoint. This field is reflected in the PSC Forwarding Rule configuration to enable global access.
+    values:
+      - :GLOBAL
+      - :REGIONAL
+  - !ruby/object:Api::Type::String
+    name: 'pscForwardingRule'
+    output: true
+    description: |
+      The resource reference of the PSC Forwarding Rule created on behalf of the customer. Format: `//compute.googleapis.com/projects/{project}/regions/{region}/forwardingRules/{forwarding_rule_name}`
+  - !ruby/object:Api::Type::String
+    name: 'address'
+    description: |
```

Comment:
Can you please add a note here that the users may need to specify the address value rather than use a reference?

Reply:
We accept both reference and IP literal. I added a explanatory note under description.